internal/display/roles.go

```diff
@@ -58,6 +58,24 @@ func (r *Renderer) RoleList(roles []*management.Role) {
 	r.Results(res)
 }
 
+func (r *Renderer) UserRoleList(roles []*management.Role) {
+	resource := "roles assigned to user"
+	r.Heading(fmt.Sprintf("%s (%d)", resource, len(roles)))
+
+	if len(roles) == 0 {
+		r.EmptyState(resource)
```

Comment:
When printed, this empty state doesn't read correctly. It states: "No roles assigned to user available". Instead, this should just be "No roles assigned to user".

Reply:
The `EmptyState()` func is used across 14 different commands and it looks like:

```go
func (r *Renderer) EmptyState(resource string) {
	fmt.Fprintf(r.MessageWriter, "No %s available.\n\n", resource)
}
```

If we remove the "available" part the result will look like:

```
No actions.
No apis. 
No scopes.
No applications.
No custom domains.
No user. 
... etc.
```

What would be something that would read better for all of them? 🤔

Reply:
Update: We agreed to not use the `EmptyState` func for this one.